variables.tf

```diff
@@ -125,3 +125,9 @@ variable "remote_metastores" {
   type        = "list"
   default     = []
 }
+
+variable "enable_remote_metastore_dns" {
+  description = "Option to enable creating dns records for remote metastores"
```

Comment:
dns -> DNS
I think we're now ending all our "description" entries with a "." so they show up nicely in the documentation.